mbuild/tests/test_packing.py

```diff
@@ -36,6 +36,16 @@ def test_fill_region_multiple(self, ethane, h2o):
         assert np.max(filled.xyz[:16, 0]) < 4
         assert np.min(filled.xyz[16:, 0]) > 4
```

Comment:
Looks good, would be useful to have a list of multiple types for the region command (list, Box, etc.)

Could you write up a small test with multiple input regions to ensure that is also correct?